### ISSUE TITLE
Popup Login for Platform

### ISSUE THREAD

**Issue by nitin10s** [CircuitVerse/CircuitVerse#1421]
**Is your feature request related to a problem? Please describe.**
As discussed over Slack to change the login screen to a pop-up since it also gets accessed from the Simulator. 

![image](https://user-images.githubusercontent.com/16988558/81468709-d5a06200-91fe-11ea-9527-bbf3ce3d7819.png)
![image](https://user-images.githubusercontent.com/16988558/81468711-d933e900-91fe-11ea-98bd-893fef6ac9fc.png)

**Comment by Kushagraw12** [comment]
@nitin10s  Can I work on this issue?
Thanks!

**Comment by nitin10s** [comment]
@aayush-05

**Comment by nitin10s** [comment]
@Kushagraw12 You may also look at https://github.com/CircuitVerse/blog/issues/5

**Comment by nitin10s** [comment]
Also, check out issues in our Interactive Book and Docs repo. The main platform is currently being worked upon by @aayush-05 as a GSOC project.

**Comment by aayush-05** [comment]
> @aayush-05

Got done with Multiple mentors functionality backend; will give this one a shot before moving on to GDPR

**Comment by nitin10s** [comment]
No, I meant can you please point @Kushagraw12 to some relevant issues.

**Comment by Kushagraw12** [comment]
> No, I meant can you please point @Kushagraw12 to some relevant issues.

Yeah please, I am new to this org, and looking forward to making contributions! 
I looked at the issue you pointed me to. Can I give it shot?
Thanks!

**Comment by aayush-05** [comment]
Sure @Kushagraw12, you are free to take any issue (just make sure someone hasn't commented on that before).

**Comment by Kushagraw12** [comment]
Okay thanks @aayush-05 ! I will take a look at them and try to solve them. 
Thanks :)

**Comment by aayush-05** [comment]
Great, ping me on Slack if you need any help

**Comment by vedant-jain03** [comment]
@nitin10s is this issue is open now?

**Comment by Kartikay304** [comment]
Working in this issue

**Comment by Me-Priyank** [comment]
Already covered, superseded by JWT auth architecture.
